tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -386,111 +386,6 @@ public function test_get_settings_appearance_false_does_not_opt_in() {
 		$this->assertEqualSetsWithIndex( $expected, $actual );
 	}
 
-	/**
```

Comment:
It looks like this is missing the `data_get_block_style_variation_selector` and `test_get_block_style_variation_selector` from Gutenberg

Reply:
Yeah, I noted that in the other PR. They were only merged a few days ago in https://github.com/WordPress/gutenberg/pull/58051, so I figured there would be a separate backport for them. The tests weren't passing here, so I figured they needed some additional PHP backports to pass.

Reply:
> The tests weren't passing here, so I figured they needed some additional PHP backports that I wasn't going to add here.

It makes sense they'd fail as the tests cover a new function that was added and used within the theme.json class.

A backport is now up (sorry for the delay) in https://github.com/WordPress/wordpress-develop/pull/6053.